src/core/models.py

```diff
@@ -1344,9 +1344,16 @@ class Meta:
         max_length=15,
         choices=review_type_choices(),
     )
+    # TODO: refactor 'user' to 'reviewer'
```

Comment:
PyCharm's refactoring tool wasn't able to change `ReviewAssignment.user`' to `ReviewAssignment.reviewer`. I'm assuming this is because `user` is quite a common identifier.

I can get rid of the TODO as I can think of no way of actually refactoring it. It just annoys me that they've used the type of the reference and not a more descriptive name.